docs/docs/develop/policies.md

```diff
@@ -0,0 +1,118 @@
+- `admin` - whether the user is an admin or not. Example - `true`
+
+User attributes will be namespaced with `.user` prefix while being used, more in the example below
+ 
+## Examples
+
+1. Let's say you want to restrict access to a dashboard only to admin users. You can define the following policy in the `dashboard_name.yaml` file under `policy` section.
+    ```yaml 
+    policy:
+      has_access: "'{{ .user.admin }}' == true"
+    ```
+   > **_Note:_**  If `policy` section is defined and `has_access` is not, then it will resolve to `false` meaning dashboard won't be accessible to anyone.
+2. Along with the above policy, you also want to filter all counts/measures as per the user domain. Assuming you have a `domain` dimension in your model. You can define the following policy in the `dashboard_name.yaml` file under `policy` section.
+    ```yaml
+    policy:
+      has_access: "'{{ .user.admin }}' == true"
+      filter: "domain = '{{ .user.domain }}'"
+     ```
+   > **_Note:_**  `filter` condition needs to be a valid SQL `WHERE` clause. It will be injected in the queries after resolving the user attributes templating.
+3. Along with the above policy, you want to exclude certain dimension for user not having `test.com` domain. Assuming you have a `ssn` dimension in your model. You can define the following policy in the `dashboard_name.yaml` file under `policy` section.
+    ```yaml
+    policy:
+      has_access: "'{{ .user.admin }}' == true"
+      filter: "domain = '{{ .user.domain }}'"
+      exclude:
+        - name: ssn
+          if: "{{ .user.domain }} != 'test.com'"
+    ```
+   As per above ssn dimension will be excluded but all other dimensions will be included.
+4. Here's one example for column inclusion.
+
+    ```yaml
+    policy:
+      has_access: "true"
+      filter: "domain = '{{ .user.domain }}'"
+      include:
+        - name: ssn
+          if: "'{{ .user.admin }}' == true"
+    ```
+   > **_Note:_** If include is defined all other dimensions will be excluded. Only one of the
+     `include`/`exclude` can be defined at a time.
+
+> **_Note:_** `has_access` and `if` conditions on `include`/`exclude` fields needs to be valid expressions that evaluates to true/false.
+
+## Advanced Examples
+Policies are resolved in two phases - 
+1. The templating engine replaces placeholder like this `{{ }}` with actual values
+2. For `has_access` and `if` conditions, the resultant expression is evaluated to true/false, whereas for `filter` condition the resultant expression is injected in the query.
+
+> **_Note:_** We use sprig templating engine for templating and govaluate for expression evaluation. So any valid operator/expression they support will work here. You can refer to their documentation for more details.
+[Here](http://masterminds.github.io/sprig/) are the supported sprig functions and [here](https://github.com/Knetic/govaluate/blob/master/MANUAL.md#operators) are the supported expression evaluation operators. 
```

Comment:
We actually use the Go [`text/template`](https://pkg.go.dev/text/template) templating engine with Sprig functions added to the context.